src/AgGrid.elm

```diff
@@ -251,6 +252,8 @@ type alias GridConfig =
     , size : String
     , suppressMenuHide : Bool
     , themeClasses : Maybe String
+    , detailRenderer : Maybe { componentName : String, componentParams : Maybe Json.Encode.Value }
```

Comment:
Let's try to keep values sorted. It's always helpful, especially when things get extended more and more.